src/machine/machine_rp2_usb.go

```diff
@@ -149,6 +162,37 @@ func sendStallViaEPIn(ep uint32) {
 	_usbDPSRAM.EPxBufferControl[ep&0x7F].In.Set(val)
 }
 
+// Enable ENDPOINT_HALT/stall on a USB OUT endpoint
```

Comment:
Do you mean `Enable` and not `Set` (as in the name) or `Flag`? As far a I understood your description, this is called to tell the hardware and/or the peer that there is a stall. If so, `Enable` contributed to my initial confusion about the purpose of the function.

Nit: end documentation comments with periods.

Reply:
I think of them as synonymous, but sure if it's generally more clear I can update it